coordinator/src/services/translation/common/index.ts

```diff
@@ -179,7 +179,7 @@ export function wrapInOperationOutcome<T>(message: SpineDirectResponse<T>): fhir
     issue: [{
       code: message.statusCode <= 299 ? "informational" : "invalid",
       severity: message.statusCode <= 299 ? "information" : "error",
-      diagnostics: message.body.toString()
+      diagnostics: message.body?.toString()
```

Comment:
When receiving timeout responses from sync-wrap the response body is null so without this our api returns server errors